.github/workflows/commit-workflow.yml

```diff
@@ -105,6 +131,27 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Enable cache
+        # https://github.com/marketplace/actions/rust-cache
+        uses: Swatinem/rust-cache@v1
+
+      # In usual case, we don't need this step.
+      # There is an issue with cargo-outdated 0.10.0, therefore, explicitly need to install the latest version.
+      # This step takes a certain time, so it should be removed in the future.
+      - name: Cargo install outdated
+        uses: actions-rs/cargo@v1
+        # https://github.com/marketplace/actions/rust-cargo
+        with:
+          command: install
+          args: cargo-outdated
```

Comment:
Don't we need to specify which version needs to be installed?. Will this always pull and install the recent version?